dockerfile.dev

```diff
@@ -0,0 +1,26 @@
+# Use the official python 2 base image
+FROM python:2
+
```

Comment:
Set up a environment variable to avoid writing pyc files?

Reply:
@Ugtan Any reason why to avoid writing pyc files in a container?

Reply:
To reduce the image size. Maybe you can try adding it to see it makes any difference in the overall size of the image?

Reply:
I have added it to docker-compose so users can have it on/off on their preference. 
Also added `.dockerignore` so cache, test files, git files, and other such do not creep into the image.

Reply:
Great!